code/modules/awaymissions/mission_code/ghost_role_spawners/golems.dm

```diff
@@ -134,6 +134,27 @@
 		user.death()
 		return
 
+/obj/effect/mob_spawn/human/golem/attackby(obj/item/slimepotion/transference/I, mob/living/carbon/user, params)
+	. = ..()
+	if(!istype(I))
+		return
+	if(istype(src, /obj/effect/mob_spawn/human/golem/servant))
+		has_owner = FALSE
+	flavour_text = null
+	if(iscarbon(user) && can_transfer)
+		var/human_transfer_choice = alert("Transfer your soul to [src]? (Warning, your old body will die!)",null,"Yes","No")
+		if(human_transfer_choice != "Yes")
+			return
+		if(QDELETED(src) || uses <= 0)
+			return
+		user.visible_message("<span class='notice'>As [user] applies the potion on the golem shell, a faint light leaves them, moving to [src] and animating it!</span>",
+		"<span class='notice'>You apply the potion to [src], feeling your mind leave your body!</span>")
+		message_admins("[key_name(user)] used [I] to transfer their mind into [src]")
+		create(ckey = user.ckey, name = user.real_name)
+		user.death()
+		QDEL_NULL(user.internal_organs_slot[BRAIN])
```

Comment:
Brutal. Won't this negatively affect antagonists with a debrain objective?

Reply:
The alternative is (at least according to SabreML): dusting them (which prevents them from getting their ID or any of their other gear), or letting them just die (which theoretically makes the old body revivable)

Reply:
The other problem is that adjusting this to make the mind transfer would require a rework of the create() proc in and of itself.

Reply:
The old body would be revivable through Strange Reagent, and it would result in a catatonic body since the mind was transferred already.

Reply:
To be honest, I have no idea how to go about resolving this.

Reply:
What I mean to say is, I think it's fine if the old body stays intact. Even if it gets revived, they wouldn't be controlled so it hardly matters. If anything, it could help anyone looking for the potion user since they aren't dusted.

Reply:
I hadn't considered antag objectives when I suggested those, but it would probably be better to just leave the body as-is like Mochi suggested.
Any potential weirdness from a catatonic clone is less important than accidentally forcing redtext on a changeling.

Reply:
I think it's settled than that there will be no brain deletion.